contracts/market/FifoTokenizedCommodityMarket.sol

```diff
@@ -15,35 +18,35 @@ contract FifoTokenizedCommodityMarket is StandardTokenizedCommodityMarket, IEIP7
   }
 
 
-  function getEarliestSale() public view returns (uint, uint) {   
-    if (commoditiesForSale.length >= 0) {    
-      for (uint i = 0; i < commoditiesForSale.length; i++ ) {
+  function getEarliestSale() public view returns (uint, uint) {
+    if (commoditiesForSale.length >= 0) {
+      for (uint i = 0; i < commoditiesForSale.length; i = i.add(1) ) {
```

Comment:
we should also explicitly change uint to be `uint256`, in  that case using ++ shouldn't result in an overflow anyways, but I guess it doesn't hurt to have uint256 explicitly AND safemath